src/wallet/feebumper.cpp

```diff
@@ -218,9 +220,9 @@ Result CreateRateBumpTransaction(CWallet& wallet, const uint256& txid, const CCo
     CTransactionRef tx_new = MakeTransactionRef();
     CAmount fee_ret;
     int change_pos_in_out = -1; // No requested location for change
-    std::string fail_reason;
+    bilingual_str fail_reason;
     if (!wallet.CreateTransaction(*locked_chain, recipients, tx_new, fee_ret, change_pos_in_out, fail_reason, new_coin_control, false)) {
-        errors.push_back("Unable to create transaction: " + fail_reason);
+        errors.push_back(_("Unable to create transaction.") + fail_reason);
```

Comment:
fa5a95c678ba2a4ebdbaad084a9fe18ba5b64629
Hereinafter, it should looks like https://github.com/bitcoin/bitcoin/blob/fa5a95c678ba2a4ebdbaad084a9fe18ba5b64629/src/wallet/wallet.cpp#L199

Reply:
Thanks, good catch.